frontend/src/App.tsx

```diff
@@ -44,7 +44,7 @@ const App = () => {
   }, []);
 
   const isInitialized = () => {
-    return !keycloak?.authenticated || keycloak?.userInfo || keycloakUserLoaded;
+    return keycloak && keycloakUserLoaded;
```

Comment:
I guess this fixes the login issues in DEV?

Reply:
unrelated, the change above shouldn't affect behaviour, it just remove some redundant checks.